app/web/features/profile/view/leaveReference/formSteps/Appropriate.tsx

```diff
@@ -42,7 +42,12 @@ export default function Appropriate({
   const theme = useTheme();
```

Comment:
Not sure if we should just move the error next to the Q?

![Screenshot from 2024-12-12 13-47-22](https://github.com/user-attachments/assets/be4eeb77-3ab3-4a86-b06a-4bd2b69053ab)

Reply:
Done